components/gene-panels/api/src/main/java/org/phenotips/panels/GeneCountsDto.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+package org.phenotips.panels;
+
+import org.xwiki.stability.Unstable;
+
+import java.util.Set;
+
+/**
+ * A DTO containing a set of phenotypes associated with a specific gene, as well as the number of phenotypes.
+ *
+ * @version $Id$
+ * @since 1.3M5
+ */
+@Unstable("New API introduced in 1.3")
+public interface GeneCountsDto
+{
+    /**
+     * Gets the total size of the set of HPO terms.
+     *
+     * @return the total size of the HPO terms set
+     */
+    int getCount();
+
+    /**
+     * Gets the set of HPO terms associated with a specific gene.
+     *
+     * @return a set of HPO terms associated with a specific gene
+     */
+    Set<String> getHpoTerms();
+
+    /**
+     * Adds the HPO term to the set.
+     *
+     * @param hpoTerm the HPO term to be added
+     */
+    void addHpoTerm(final String hpoTerm);
```

Comment:
I'd suggest this be just `addTerm(termId)` and make the code as HPO-agnostic as possible.